### DISCUSSION TITLE
想提升模型翻译能力，双语语料应该精调还是预训练？

### DISCUSSION THREAD

**Discussion by ovo-Tim** [ymcui/Chinese-LLaMA-Alpaca-2#401] [Q&A]
想用来翻译一些教程视频，但是希望可以进一步提升英译中的能力，应该把已有的双语语料放入预训练数据集中还是进行指令精调？🙂

**Comment by ymcui** [comment]
不一定是二选一，你也可以尝试两个阶段都加。

**Reply by ovo-Tim** [reply]
感谢您的回复。
我不知道我们是否还缺少平行语料的训练，如果缺少还需要多少？(根据文章，过量的平行语料适得其反)

或者说我们不再需要平行语料了，仅仅是单语训练就足够了？🤔

**Reply by ovo-Tim** [reply]
正如我在 https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/discussions/403 当中的提问
